ui/client/http/HttpService.js

```diff
@@ -373,12 +358,12 @@ export default {
       url: `${API_URL}/processes/${processId}/${processCategory}?isSubprocess=${isSubprocess}`,
       type: 'POST'
     }).then(callback, (error) => {
-      this.addError(`Failed to create process:`, error, true);
+      this.addError(`Failed to create process:`, error, true)
     })
   },
 
   importProcess(processId, file, callback, errorCallback) {
-    var formData = new FormData();
+    let formData = new FormData()
     formData.append("process", file)
 
     return ajaxCallWithoutContentType({
```

Comment:
why are we leving it here? I mean, all old ajaxCalls?

Reply:
Okey, I will change all old ajax requests..